.github/workflows/conventional-commits.yml

```diff
@@ -0,0 +1,19 @@
+# Validates PR title follows conventional commits
+name: conventional commits check
+on:
+  pull_request:
+    branches: main
+    types:
+      - edited
+      - opened
+      - synchronize
+      - reopened
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  conventional_commit_title:
+    runs-on: [ARM64, self-hosted, Linux]
+    steps:
+      - uses: chanzuckerberg/github-actions/.github/actions/conventional-commits@v2.1.0
```

Comment:
Oh this is neat! Are other engineers aware of this new requirement? I took a look at https://github.com/chanzuckerberg/github-actions/blob/main/.github/actions/conventional-commits/action.yml and it's a little different than how others have been opening PRs thus far.

Reply:
@richaagarwal Yes, the titles up until now haven't followed the conventional pattern. We would have to update the value [here](https://github.com/chanzuckerberg/cryoet-data-portal/blob/f20840721085ae5e1e1b4e9e71238f8531a04807/release-please.config.json#L2) for the commits from which we start using the change log generation. 
After this PR has been merged, I will add a message about the commit title requirements to the team's Slack channel. The check gives a clear message on how to fix the failure, which should be helpful as well. 

@codemonkey800 I think it would be good to have the title check for all the PRs in the interest of consistency.